src/index.js

```diff
@@ -1 +1,44 @@
-import "./style.css";
\ No newline at end of file
+import './style.css';
+
+class todos {
+  static displayToDo() {
+    const todo = [
+      {
+        description: 'Read a book',
+        completed: false,
+        index: 0,
+      },
+      {
+        description: 'Read the bible',
+        completed: false,
+        index: 1,
+      },
+      {
+        description: 'Write my daily plan',
+        completed: false,
+        index: 2,
+      },
+      {
+        description: 'Attend Microverse Classes',
+        completed: false,
+        index: 3,
+      },
+    ];
+    todo.forEach((item) => todos.addTodo(item));
+  }
+
+  static addTodo(item) {
+    const list = document.querySelector('.list');
+    const todoli = document.createElement('li');
+    todoli.innerHTML = `
+    <li class="listItems">
+                <input type="checkbox" name="select" id="" class="check" />
+                <p class="title">${item.description}</p>
+                <i id="moreicon" class="fa-solid fa-ellipsis-vertical"></i>
+              </li>
+    `;
+    list.appendChild(todoli);
+  }
+}
```

Comment:
- [ ] Kindly align your task description to the left instead of the center as we have on the design video shared with you.

  your css | design css
  :---:|:---:
  ![hfyeie](https://user-images.githubusercontent.com/36542986/207307130-598f5ff3-9841-4177-bbee-1b078b3f1b82.jpg) | ![kdkd](https://user-images.githubusercontent.com/36542986/207307136-3af0c761-328e-4bb9-a643-74ce4deceafd.jpg)